File: common/src/main/java/de/invees/portal/common/model/v1/section/configuration/SectionConfigurationEntryV1.java
package de.invees.portal.common.model.v1.section.configuration;

import de.invees.portal.common.model.v1.DisplayV1;
import de.invees.portal.common.model.Model;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class SectionConfigurationEntryV1 implements Model {

  private String key;
  private DisplayV1 displayName;
  private List<SectionConfigurationEntryOptionV1> optionList;

}

File: model/v1/section/configuration/SectionConfigurationEntryV1.ts
import DisplayV1 from "@/model/v1/DisplayV1.ts";
import SectionConfigurationEntryOptionV1 from "@/model/v1/section/configuration/SectionConfigurationEntryOptionV1.ts";

export default interface SectionConfigurationEntryV1 {
  key: string;
  displayName: DisplayV1;
  optionList: Array<SectionConfigurationEntryOptionV1>;
}
